.ado/publish.yml

```diff
@@ -40,15 +40,18 @@ variables:
     value: microsoft
   - name: ArtifactServices.Symbol.PAT
     value: $(pat-symbols-publish-microsoft)
+  - name: AgentPool.Medium.Microsoft 
+    value: rnw-pool-4-microsoft
+  - name: AgentPool.Large.Microsoft
+    value: rnw-pool-8-microsoft
 
 trigger: none
 pr: none
 
 jobs:
   - job: RnwNpmPublish
     displayName: React-Native-Windows Npm Build Rev Publish
-    pool:
-      vmImage: $(VmImage)
+    pool: $(AgentPool.Medium.Microsoft)
```

Comment:
Why use a new variable name instead of having different values to choose from for `$(VmImage)`?

Reply:
We haven't imported those into the publish template yet, so I didn't want to start now.
I have hopes that some time I have time to improve the publish pipeline by splitting out the build part with the CI pipeline's build outputs and then have the release definitoin take care of the git push, npm push and nuget pushes...